export docs (#4)
phero-hq/website: docs/deploying-phero/vercel.md

---
title: "Deploying to Vercel"
---

> In order to deploy your Phero server, there's one last step: making an export.
> When you export you server, Phero compiles an optimized server for you to deploy.

Reviewer: ```suggestion
In order to deploy your Phero server, there's one last step: making an export. When you export your server, Phero compiles an optimised bundle for you to deploy.
```

> For now Phero compiles your services into Vercel Serverless Functions. There's no way yet to compile it into Edge functions.

Reviewer: ```suggestion
For now, Phero compiles your services into Vercel Serverless Functions, Edge Functions are not yet supported.
```

> First navigate to your server dir:

Reviewer: ```suggestion
First, navigate to your server directory:
```

```
cd api
```

To export your Phero server, run:

```
npx phero@latest server export --flavor vercel
```

> This will generate an hidden `.vercel` directory with all the code necessary to deploy.

Reviewer: ```suggestion
This will generate a `.vercel` directory, with all the code necessary to deploy.
```

> To deploy, please run:

Reviewer: ```suggestion
To deploy this bundle, run:
```

Author: hmmm bundle could be a bit misleading as we're deploying multiple functions/bundles..

Reviewer: "To deploy your server"?

```
  npx vercel@latest deploy --prebuilt
```

> That's it!
> 
> For each subsequent deploy you can repeat the steps above.

Reviewer: ```suggestion
That's it! For each subsequent deploy you can repeat the steps above.
```
